# DOB Job Applications Filings Job Market Data Analysis

- This notebook is an exploration of the NYC construction job market in the time period of 2019-2021.
- The data has been provided by the NYC Open Data collection of public databases. 
- Each row of data in this initial file will represent a construction job that has been fully permitted within this time period and is included in the job type category "NB" which denotes that the job is a new building. 
- Throughout the notebook, further filters will be applied to this dataset to analyse the job market to compare jobs that are between 7 and 9 stories and jobs that are 10 stories or above.

In [98]:
import pandas as pd

In [99]:
data_df = pd.read_csv("DOB_Job_App_Filings_2019_2021.csv", sep = ',', engine = 'python')

In [100]:
pd.set_option('display.max_columns', None)
pd.set_option('display.max_rows', None)

# Total Job Market 2019-2021

In [101]:
data_df.head()

Unnamed: 0      Job #    Borough House #         Street Name Job Type  \
0           0  220538611      BRONX     915          ELSMERE PL       NB   
1           1  220152803      BRONX    1325  SOUTHERN BOULEVARD       NB   
2           2  220649706      BRONX    3923       CARPENTER AVE       NB   
3           3  121205604  MANHATTAN     270         PARK AVENUE       NB   
4           4  321597795   BROOKLYN     963     ATLANTIC AVENUE       NB   

  Job Status                Job Status Descrp Latest Action Date  \
0          R  PERMIT ISSUED - ENTIRE JOB/WORK         11/05/2021   
1          R  PERMIT ISSUED - ENTIRE JOB/WORK         11/15/2021   
2          R  PERMIT ISSUED - ENTIRE JOB/WORK         04/29/2021   
3          R  PERMIT ISSUED - ENTIRE JOB/WORK         11/30/2021   
4          R  PERMIT ISSUED - ENTIRE JOB/WORK         06/22/2021   

  Building Type  Community - Board Applicant's First Name  \
0        OTHERS                206                 MINKUK   
1        OTHERS                203                SUZANNA   
2        OTHERS                212                  JULIO   
3        OTHERS                105               NICKOLAS   
4        OTHERS                302                 DANIEL   

  Applicant's Last Name Applicant Professional Title  Applicant License #  \
0                   KIM                           RA                32897   
1               THARIAN                           RA                29684   
2            LEDER-LUIS                           RA                20478   
3             ZIGOMANIS                           RA                33081   
4             BERNSTEIN                           RA                16854   

  Professional Cert Fully Permitted  Existing Zoning Sqft  \
0                 N      2021-04-29                     0   
1                 N      2019-03-07                     0   
2                 N      2021-03-16                     0   
3                 N      2021-11-30                     0   
4                 N      2021-06-22                     0   

   Proposed Zoning Sqft  ExistingNo. of Stories  Proposed No. of Stories  \
0                 49075                       0                        9   
1                 34363                       0                        8   
2                 20237                       0                        7   
3               1864801                       0                       57   
4                 88368                       0                        9   

   Existing Height  Proposed Height Owner's First Name Owner's Last Name  \
0                0               90                JAY           MARTINO   
1                0               85           NORBERTO         VELAZQUEZ   
2                0               65             ROBERT             LUMAJ   
3                0             1388              DEVIN             MAHER   
4                0               95             EFRAIM             WEBER   

            Owner's Business Name  City   Owner'sPhone #  \
0                     STAGG GROUP    NaN      9146687511   
1       1319-25 SOUTHERN BLVD LLC    NaN      3475655481   
2        ATLANTIS RENOVATIONS INC    NaN      3476210292   
3  JP MORGAN AND CHASE BANK, N.A.    NaN      2122704958   
4                86 LEFFERTS CORP    NaN      9176824871   

                             Job Description           DOBRunDate  JOB_S1_NO  \
0  ERECTION OF 9 STORY RESIDENTIAL BUILDING.  11/23/2021 00:00:00    2637000   
1                                        NaN  11/16/2021 00:00:00    2802048   
2                                        NaN  04/30/2021 00:00:00    2965629   
3                                        NaN  12/09/2021 00:00:00    3126397   
4                                        NaN  10/12/2021 00:00:00    3158207   

   GIS_LATITUDE  GIS_LONGITUDE              GIS_NTA_NAME  
0     40.841590     -73.885100              East Tremont  
1     40.831084     -73.891496         Crotona Park East  
2     4

In [102]:
df = data_df.drop(columns=['Unnamed: 0'])

In [103]:
df.head()

Job #    Borough House #         Street Name Job Type Job Status  \
0  220538611      BRONX     915          ELSMERE PL       NB          R   
1  220152803      BRONX    1325  SOUTHERN BOULEVARD       NB          R   
2  220649706      BRONX    3923       CARPENTER AVE       NB          R   
3  121205604  MANHATTAN     270         PARK AVENUE       NB          R   
4  321597795   BROOKLYN     963     ATLANTIC AVENUE       NB          R   

                 Job Status Descrp Latest Action Date Building Type  \
0  PERMIT ISSUED - ENTIRE JOB/WORK         11/05/2021        OTHERS   
1  PERMIT ISSUED - ENTIRE JOB/WORK         11/15/2021        OTHERS   
2  PERMIT ISSUED - ENTIRE JOB/WORK         04/29/2021        OTHERS   
3  PERMIT ISSUED - ENTIRE JOB/WORK         11/30/2021        OTHERS   
4  PERMIT ISSUED - ENTIRE JOB/WORK         06/22/2021        OTHERS   

   Community - Board Applicant's First Name Applicant's Last Name  \
0                206                 MINKUK                   KIM   
1                203                SUZANNA               THARIAN   
2                212                  JULIO            LEDER-LUIS   
3                105               NICKOLAS             ZIGOMANIS   
4                302                 DANIEL             BERNSTEIN   

  Applicant Professional Title  Applicant License # Professional Cert  \
0                           RA                32897                 N   
1                           RA                29684                 N   
2                           RA                20478                 N   
3                           RA                33081                 N   
4                           RA                16854                 N   

  Fully Permitted  Existing Zoning Sqft  Proposed Zoning Sqft  \
0      2021-04-29                     0                 49075   
1      2019-03-07                     0                 34363   
2      2021-03-16                     0                 20237   
3      2021-11-30                     0               1864801   
4      2021-06-22                     0                 88368   

   ExistingNo. of Stories  Proposed No. of Stories  Existing Height  \
0                       0                        9                0   
1                       0                        8                0   
2                       0                        7                0   
3                       0                       57                0   
4                       0                        9                0   

   Proposed Height Owner's First Name Owner's Last Name  \
0               90                JAY           MARTINO   
1               85           NORBERTO         VELAZQUEZ   
2               65             ROBERT             LUMAJ   
3             1388              DEVIN             MAHER   
4               95             EFRAIM             WEBER   

            Owner's Business Name  City   Owner'sPhone #  \
0                     STAGG GROUP    NaN      9146687511   
1       1319-25 SOUTHERN BLVD LLC    NaN      3475655481   
2        ATLANTIS RENOVATIONS INC    NaN      3476210292   
3  JP MORGAN AND CHASE BANK, N.A.    NaN      2122704958   
4                86 LEFFERTS CORP    NaN      9176824871   

                             Job Description           DOBRunDate  JOB_S1_NO  \
0  ERECTION OF 9 STORY RESIDENTIAL BUILDING.  11/23/2021 00:00:00    2637000   
1                                        NaN  11/16/2021 00:00:00    2802048   
2                                        NaN  04/30/2021 00:00:00    2965629   
3                                        NaN  12/09/2021 00:00:00    3126397   
4                                        NaN  10/12/2021 00:00:00    3158207   

   GIS_LATITUDE  GIS_LONGITUDE              GIS_NTA_NAME  
0     40.841590     -73.885100              East Tremont  
1     40.831084     -73.891496         Crotona Park East  
2     40.887420     -73.864069  Williamsbridge-Olinville  
3     40.755641     -73.

In [104]:
#Checking to confirm that each of the jobs in this dataset have been fully permitted

In [105]:
df['Fully Permitted'].isnull().values.any()

False

In [106]:
#Each job has been fully permitted. 

In [107]:
#Converting the fully permitted column from a string format (text) to a datetime format, to be readable in python.

In [108]:
df['Fully Permitted'] = pd.to_datetime(df['Fully Permitted'], errors ="coerce", infer_datetime_format=True)

In [109]:
df['Fully Permitted'].iloc[1:50]

1    2019-03-07
2    2021-03-16
3    2021-11-30
4    2021-06-22
5    2021-05-05
6    2021-10-14
7    2021-08-30
8    2019-02-11
9    2021-11-09
10   2021-04-30
11   2021-08-26
12   2021-12-09
13   2021-11-17
14   2021-10-06
15   2021-08-18
16   2020-12-21
17   2021-07-29
18   2021-05-03
19   2019-12-03
20   2021-09-15
21   2021-09-16
22   2020-05-20
23   2021-05-11
24   2021-12-30
25   2019-01-07
26   2019-01-29
27   2019-08-20
28   2021-02-28
29   2020-08-26
30   2021-02-18
31   2021-07-13
32   2021-10-01
33   2019-01-22
34   2021-05-06
35   2019-11-21
36   2020-04-17
37   2019-12-10
38   2019-08-08
39   2021-06-29
40   2021-05-05
41   2021-06-25
42   2021-09-14
43   2021-12-17
44   2021-05-28
45   2021-08-16
46   2019-08-19
47   2019-12-04
48   2021-08-06
49   2021-11-12
Name: Fully Permitted, dtype: datetime64[ns]

- Now we will get a record of the total number of fully approved NB jobs there are in this time period regardless of the proposed number of floors:

In [110]:
index = df.index
number_of_rows = len(index)
print(number_of_rows)

697


In [111]:
#The dataset will now be filtered to show how many of these jobs 10 stories high or greater.

In [112]:
proposed_over_ten = df['Proposed No. of Stories'] >= 10

In [113]:
job_market_19_21_10_plus = df.loc[proposed_over_ten]

In [114]:
job_market_19_21_10_plus.head()

Job #    Borough House #     Street Name Job Type Job Status  \
3   121205604  MANHATTAN     270     PARK AVENUE       NB          R   
6   210182283      BRONX    2351   WALTON AVENUE       NB          R   
8   121187802  MANHATTAN     145      WEST 47 ST       NB          R   
10  321588545   BROOKLYN      34       UNION AVE       NB          R   
16  320978064   BROOKLYN    2286  CROPSEY AVENUE       NB          R   

                  Job Status Descrp Latest Action Date Building Type  \
3   PERMIT ISSUED - ENTIRE JOB/WORK         11/30/2021        OTHERS   
6   PERMIT ISSUED - ENTIRE JOB/WORK         10/14/2021        OTHERS   
8   PERMIT ISSUED - ENTIRE JOB/WORK         11/05/2021        OTHERS   
10  PERMIT ISSUED - ENTIRE JOB/WORK         04/30/2021        OTHERS   
16  PERMIT ISSUED - ENTIRE JOB/WORK         12/21/2020        OTHERS   

    Community - Board Applicant's First Name Applicant's Last Name  \
3                 105               NICKOLAS             ZIGOMANIS   
6                 205                  DAVID                 GROSS   
8                 105               JONATHAN                  MOSS   
10                301                 FARIBA                MAKOOI   
16                311                MICHAEL                  KANG   

   Applicant Professional Title  Applicant License # Professional Cert  \
3                            RA                33081                 N   
6                            RA                16587                 N   
8                            RA                25207                 N   
10                           RA                34686                 N   
16                           RA                24849                 N   

   Fully Permitted  Existing Zoning Sqft  Proposed Zoning Sqft  \
3       2021-11-30                     0               1864801   
6       2021-10-14                     0                 28585   
8       2019-02-11                     0                263586   
10      2021-04-30                     0                423295   
16      2020-12-21                     0                241259   

    ExistingNo. of Stories  Proposed No. of Stories  Existing Height  \
3                        0                       57                0   
6                        0                       12                0   
8                        0                       48                0   
10                       0                       12                0   
16                       0                       23                0   

    Proposed Height Owner's First Name Owner's Last Name  \
3              1388              DEVIN             MAHER   
6               114             STEVEN             MOUNT   
8               509              DAVID            WROBEL   
10              114              SIMON         DUSHINSKY   
16              240           MENG HUA              WANG   

             Owner's Business Name  City   Owner'sPhone # Job Description  \
3   JP MORGAN AND CHASE BANK, N.A.    NaN      2122704958             NaN   
6            2351 WALTON OWNER LLC    NaN      9172177359             NaN   
8                   CLARITY 47 LLC    NaN      2124218100             NaN   
10             HARRISON REALTY LLC    NaN      7182464762             NaN   
16           CROPSEY PARTNERS LLC.    NaN      7189391002             NaN   

             DOBRunDate  JOB_S1_NO  GIS_LATITUDE  GIS_LONGITUDE  \
3   12/09/2021 00:00:00    3126397     40.755641     -73.975109   
6   10/15/2021 00:00:00    3270079     40.859644     -73.902027   
8   12/28/2021 00:00:00    2765418     40.758577     -73.983100   
10  10/06/2021 00:00:00    3193905     40.700831     -73.950274   
16  05/04/2021 00:00:00    2354982     40.595843     -73.997001   

             GIS_NTA_NAME  
3   Midtown-Midtown South  
6           Fordham South  
8   Midtown-Midtown South  
10                Bedford  
16       Bensonhurst West

In [115]:
#Now we will confirm that no jobs are under 10 stories to see the unique values of this column by applying the 
#   .unique() method

In [116]:
job_market_19_21_10_plus['Proposed No. of Stories'].unique()

array([57, 12, 48, 23, 34, 13, 10, 14, 43, 16, 17, 11, 27, 31, 18, 24, 51,
       22, 15, 19, 35, 66, 26, 38, 33, 56, 29, 52, 25, 21, 20, 28, 37, 30,
       74, 60, 40, 46, 44, 32])

- Now we will calculate how many jobs of the 697 are 10 stories or higher:

In [117]:
index = job_market_19_21_10_plus.index
number_of_rows = len(index)
print(number_of_rows)

267


In [118]:
#The dataset will now be filtered to show how many NB jobs fully permitted '19-'21 are between 7 and 9 stories.

In [119]:
proposed_over_seven = df['Proposed No. of Stories'] >= 7

proposed_under_nine = df['Proposed No. of Stories'] <= 9

btwn_seven_and_nine = proposed_over_seven & proposed_under_nine

job_market_19_21_btwn_7_and_9 = df.loc[btwn_seven_and_nine]

job_market_19_21_btwn_7_and_9.head()

Job #   Borough House #         Street Name Job Type Job Status  \
0  220538611     BRONX     915          ELSMERE PL       NB          R   
1  220152803     BRONX    1325  SOUTHERN BOULEVARD       NB          R   
2  220649706     BRONX    3923       CARPENTER AVE       NB          R   
4  321597795  BROOKLYN     963     ATLANTIC AVENUE       NB          R   
5  321941182  BROOKLYN    1882            BROADWAY       NB          R   

                 Job Status Descrp Latest Action Date Building Type  \
0  PERMIT ISSUED - ENTIRE JOB/WORK         11/05/2021        OTHERS   
1  PERMIT ISSUED - ENTIRE JOB/WORK         11/15/2021        OTHERS   
2  PERMIT ISSUED - ENTIRE JOB/WORK         04/29/2021        OTHERS   
4  PERMIT ISSUED - ENTIRE JOB/WORK         06/22/2021        OTHERS   
5  PERMIT ISSUED - ENTIRE JOB/WORK         05/05/2021        OTHERS   

   Community - Board Applicant's First Name Applicant's Last Name  \
0                206                 MINKUK                   KIM   
1                203                SUZANNA               THARIAN   
2                212                  JULIO            LEDER-LUIS   
4                302                 DANIEL             BERNSTEIN   
5                316                JEFFREY                 KAMEN   

  Applicant Professional Title  Applicant License # Professional Cert  \
0                           RA                32897                 N   
1                           RA                29684                 N   
2                           RA                20478                 N   
4                           RA                16854                 N   
5                           RA                23279                 N   

  Fully Permitted  Existing Zoning Sqft  Proposed Zoning Sqft  \
0      2021-04-29                     0                 49075   
1      2019-03-07                     0                 34363   
2      2021-03-16                     0                 20237   
4      2021-06-22                     0                 88368   
5      2021-05-05                     0                 63644   

   ExistingNo. of Stories  Proposed No. of Stories  Existing Height  \
0                       0                        9                0   
1                       0                        8                0   
2                       0                        7                0   
4                       0                        9                0   
5                       0                        7                0   

   Proposed Height Owner's First Name Owner's Last Name  \
0               90                JAY           MARTINO   
1               85           NORBERTO         VELAZQUEZ   
2               65             ROBERT             LUMAJ   
4               95             EFRAIM             WEBER   
5               70            YISROEL         GREENFELD   

       Owner's Business Name  City   Owner'sPhone #  \
0                STAGG GROUP    NaN      9146687511   
1  1319-25 SOUTHERN BLVD LLC    NaN      3475655481   
2   ATLANTIS RENOVATIONS INC    NaN      3476210292   
4           86 LEFFERTS CORP    NaN      9176824871   
5          BROADWAY 1884 LLC    NaN      3476755480   

                             Job Description           DOBRunDate  JOB_S1_NO  \
0  ERECTION OF 9 STORY RESIDENTIAL BUILDING.  11/23/2021 00:00:00    2637000   
1                                        NaN  11/16/2021 00:00:00    2802048   
2                                        NaN  04/30/2021 00:00:00    2965629   
4                                        NaN  10/12/2021 00:00:00    3158207   
5                                        NaN  01/05/2022 00:00:00    3161454   

   GIS_LATITUDE  GIS_LONGITUDE              GIS_NTA_NAME  
0     40.841590     -73.885100              East Tremont  
1     40.831084     -73.891496         Crotona Park East  
2     40.887420     -73.864069  Williamsbridge-Olinville  
4     40.680263     -73.960337              Clinton Hill  
5

- Now we will calculate how many jobs of the 697 are between 7 and 9 stories high:

In [120]:
index = job_market_19_21_btwn_7_and_9.index
number_of_rows = len(index)
print(number_of_rows)

430


In [121]:
#Now we will confirm that no jobs are out of this range to see the unique values of this column by applying the 
#   .unique() method

In [122]:
job_market_19_21_btwn_7_and_9['Proposed No. of Stories'].unique()

array([9, 8, 7])

In [123]:
#To get a better breakdown of these numbers we will filter the data by the individual years in our timeframe
#and repeat the process from above to calculate the increase of market share for each year. 

- In 2019-2021, there were a total of 697 fully permitted jobs designated as new buildings ("NB")
- 267 of those jobs were 10 stories or higher
- 430 of those jobs were between 7 and 9 stories high.

# Job Market 2019

In [124]:
nineteen_start_date = "2019-01-01"

nineteen_end_date = "2019-12-31"

after_nineteen_start_date = df["Fully Permitted"] >= nineteen_start_date

before_nineteen_end_date = df["Fully Permitted"] <= nineteen_end_date

nineteen_full_year = after_nineteen_start_date & before_nineteen_end_date

nineteen_job_market = df.loc[nineteen_full_year]

nineteen_job_market.head()

Job #    Borough House #         Street Name Job Type Job Status  \
1   220152803      BRONX    1325  SOUTHERN BOULEVARD       NB          R   
8   121187802  MANHATTAN     145          WEST 47 ST       NB          R   
19  220617205      BRONX    2065         RYER AVENUE       NB          R   
25  121190861  MANHATTAN     320     WEST 135 STREET       NB          R   
26  123178499  MANHATTAN     199     CHRYSTIE STREET       NB          R   

                  Job Status Descrp Latest Action Date Building Type  \
1   PERMIT ISSUED - ENTIRE JOB/WORK         11/15/2021        OTHERS   
8   PERMIT ISSUED - ENTIRE JOB/WORK         11/05/2021        OTHERS   
19  PERMIT ISSUED - ENTIRE JOB/WORK         08/20/2021        OTHERS   
25  PERMIT ISSUED - ENTIRE JOB/WORK         12/23/2019        OTHERS   
26  PERMIT ISSUED - ENTIRE JOB/WORK         12/01/2021        OTHERS   

    Community - Board Applicant's First Name Applicant's Last Name  \
1                 203                SUZANNA               THARIAN   
8                 105               JONATHAN                  MOSS   
19                205                   ADAM                SNYDER   
25                110                  DAVID               JIMENEZ   
26                103                   WADE               JOHNSON   

   Applicant Professional Title  Applicant License # Professional Cert  \
1                            RA                29684                 N   
8                            RA                25207                 N   
19                           RA                31777                 N   
25                           RA                22841                 N   
26                           RA                19883                 N   

   Fully Permitted  Existing Zoning Sqft  Proposed Zoning Sqft  \
1       2019-03-07                     0                 34363   
8       2019-02-11                     0                263586   
19      2019-12-03                     0                 43963   
25      2019-01-07                     0                 17020   
26      2019-01-29                     0                 43147   

    ExistingNo. of Stories  Proposed No. of Stories  Existing Height  \
1                        0                        8                0   
8                        0                       48                0   
19                       0                       12                0   
25                       0                        9                0   
26                       0                       14                0   

    Proposed Height Owner's First Name Owner's Last Name  \
1                85           NORBERTO         VELAZQUEZ   
8               509              DAVID            WROBEL   
19              112               GZIM       HASANDJEKIC   
25               93             RAFAEL        SANTANDREU   
26              145              SCOTT          KUMMINGS   

               Owner's Business Name  City   Owner'sPhone #  \
1          1319-25 SOUTHERN BLVD LLC    NaN      3475655481   
8                     CLARITY 47 LLC    NaN      2124218100   
19          2065 PROPERTY OWNERS LLC    NaN      2126794590   
25              BOC ST. NICHOLAS LLC    NaN      6462256672   
26  SAGAMORE AK PROPERTY COMPANY LLC    NaN      2124526077   

            Job Description           DOBRunDate  JOB_S1_NO  GIS_LATITUDE  \
1                       NaN  11/16/2021 00:00:00    2802048     40.831084   
8                       NaN  12/28/2021 00:00:00    2765418     40.758577   
19                      NaN  08/21/2021 00:00:00    2858979     40.852175   
25                      NaN  05/06/2021 00:00:00    2638655     40.816771   
26  NEW 14TH STORY BUILDING  12/02/2021 00:00:00    2877572     40.722048   

    GIS_LONGITUDE                       GIS_NTA_NAME  
1      -73.891496                  Crotona Park East  
8      -73.983100              Midtown-Midtown South  
19     -73.901116                         Mount Hope  

In [125]:
nineteen_job_market["Fully Permitted"].max()

Timestamp('2019-12-30 00:00:00')

In [126]:
nineteen_job_market["Fully Permitted"].min()

Timestamp('2019-01-02 00:00:00')

In [127]:
index = nineteen_job_market.index
number_of_rows = len(index)
print(number_of_rows)

248


In [128]:
import geopandas as gpd
from shapely.geometry import Point, Polygon
import matplotlib.pyplot as plt

In [129]:
street_map = gpd.read_file('geo_export_9d3a09f4-3d5a-46fd-9eb8-1f37f9ef7177.shp')

In [130]:
import pyproj
from pyproj import CRS
from pyproj import Proj

In [131]:
# designate coordinate system
crs=CRS('EPSG:4326')
# zip x and y coordinates into single feature
geometry = [Point(xy) for xy in zip(nineteen_job_market['GIS_LONGITUDE'], nineteen_job_market['GIS_LATITUDE'])]
# create GeoPandas dataframe
geo_nineteen_job_market = gpd.GeoDataFrame(nineteen_job_market,
 crs = crs,
 geometry = geometry)

In [132]:
geo_nineteen_job_market.head()

Job #    Borough House #         Street Name Job Type Job Status  \
1   220152803      BRONX    1325  SOUTHERN BOULEVARD       NB          R   
8   121187802  MANHATTAN     145          WEST 47 ST       NB          R   
19  220617205      BRONX    2065         RYER AVENUE       NB          R   
25  121190861  MANHATTAN     320     WEST 135 STREET       NB          R   
26  123178499  MANHATTAN     199     CHRYSTIE STREET       NB          R   

                  Job Status Descrp Latest Action Date Building Type  \
1   PERMIT ISSUED - ENTIRE JOB/WORK         11/15/2021        OTHERS   
8   PERMIT ISSUED - ENTIRE JOB/WORK         11/05/2021        OTHERS   
19  PERMIT ISSUED - ENTIRE JOB/WORK         08/20/2021        OTHERS   
25  PERMIT ISSUED - ENTIRE JOB/WORK         12/23/2019        OTHERS   
26  PERMIT ISSUED - ENTIRE JOB/WORK         12/01/2021        OTHERS   

    Community - Board Applicant's First Name Applicant's Last Name  \
1                 203                SUZANNA               THARIAN   
8                 105               JONATHAN                  MOSS   
19                205                   ADAM                SNYDER   
25                110                  DAVID               JIMENEZ   
26                103                   WADE               JOHNSON   

   Applicant Professional Title  Applicant License # Professional Cert  \
1                            RA                29684                 N   
8                            RA                25207                 N   
19                           RA                31777                 N   
25                           RA                22841                 N   
26                           RA                19883                 N   

   Fully Permitted  Existing Zoning Sqft  Proposed Zoning Sqft  \
1       2019-03-07                     0                 34363   
8       2019-02-11                     0                263586   
19      2019-12-03                     0                 43963   
25      2019-01-07                     0                 17020   
26      2019-01-29                     0                 43147   

    ExistingNo. of Stories  Proposed No. of Stories  Existing Height  \
1                        0                        8                0   
8                        0                       48                0   
19                       0                       12                0   
25                       0                        9                0   
26                       0                       14                0   

    Proposed Height Owner's First Name Owner's Last Name  \
1                85           NORBERTO         VELAZQUEZ   
8               509              DAVID            WROBEL   
19              112               GZIM       HASANDJEKIC   
25               93             RAFAEL        SANTANDREU   
26              145              SCOTT          KUMMINGS   

               Owner's Business Name  City   Owner'sPhone #  \
1          1319-25 SOUTHERN BLVD LLC    NaN      3475655481   
8                     CLARITY 47 LLC    NaN      2124218100   
19          2065 PROPERTY OWNERS LLC    NaN      2126794590   
25              BOC ST. NICHOLAS LLC    NaN      6462256672   
26  SAGAMORE AK PROPERTY COMPANY LLC    NaN      2124526077   

            Job Description           DOBRunDate  JOB_S1_NO  GIS_LATITUDE  \
1                       NaN  11/16/2021 00:00:00    2802048     40.831084   
8                       NaN  12/28/2021 00:00:00    2765418     40.758577   
19                      NaN  08/21/2021 00:00:00    2858979     40.852175   
25                      NaN  05/06/2021 00:00:00    2638655     40.816771   
26  NEW 14TH STORY BUILDING  12/02/2021 00:00:00    2877572     40.722048   

    GIS_LONGITUDE                       GIS_NTA_NAME  \
1      -73.891496                  Crotona Park East   
8      -73.983100              Midtown-Midtown South   
19     -73.901116                         Mount Hop

In [133]:
geo_nineteen_job_market["Fully Permitted"].min()

Timestamp('2019-01-02 00:00:00')

In [135]:
geo_nineteen_job_market.to_file("geo_nineteen_job_market.geojson", driver='GeoJSON')

In [31]:
proposed_over_10_2019 = nineteen_job_market['Proposed No. of Stories'] >= 10

In [32]:
mrkt_over_10_2019 = nineteen_job_market.loc[proposed_over_10_2019]

In [33]:
mrkt_over_10_2019.head()

Job #    Borough House #        Street Name Job Type Job Status  \
8   121187802  MANHATTAN     145         WEST 47 ST       NB          R   
19  220617205      BRONX    2065        RYER AVENUE       NB          R   
26  123178499  MANHATTAN     199    CHRYSTIE STREET       NB          R   
27  210178038      BRONX    1074  WASHINGTON AVENUE       NB          R   
33  321185828   BROOKLYN     540      FULTON STREET       NB          R   

                  Job Status Descrp Latest Action Date Building Type  \
8   PERMIT ISSUED - ENTIRE JOB/WORK         11/05/2021        OTHERS   
19  PERMIT ISSUED - ENTIRE JOB/WORK         08/20/2021        OTHERS   
26  PERMIT ISSUED - ENTIRE JOB/WORK         12/01/2021        OTHERS   
27  PERMIT ISSUED - ENTIRE JOB/WORK         07/30/2021        OTHERS   
33  PERMIT ISSUED - ENTIRE JOB/WORK         05/06/2021        OTHERS   

    Community - Board Applicant's First Name Applicant's Last Name  \
8                 105               JONATHAN                  MOSS   
19                205                   ADAM                SNYDER   
26                103                   WADE               JOHNSON   
27                203                   MARK              GINSBERG   
33                302               JONATHAN                MARVEL   

   Applicant Professional Title  Applicant License # Professional Cert  \
8                            RA                25207                 N   
19                           RA                31777                 N   
26                           RA                19883                 N   
27                           RA                19534                 N   
33                           RA                21551                 N   

   Fully Permitted  Existing Zoning Sqft  Proposed Zoning Sqft  \
8       2019-02-11                     0                263586   
19      2019-12-03                     0                 43963   
26      2019-01-29                     0                 43147   
27      2019-08-20                     0                135397   
33      2019-01-22                     0                333041   

    ExistingNo. of Stories  Proposed No. of Stories  Existing Height  \
8                        0                       48                0   
19                       0                       12                0   
26                       0                       14                0   
27                       0                       12                0   
33                       0                       43                0   

    Proposed Height Owner's First Name Owner's Last Name  \
8               509              DAVID            WROBEL   
19              112               GZIM       HASANDJEKIC   
26              145              SCOTT          KUMMINGS   
27              122           SAMANTHA          MAGISTRO   
33              498             SAMUEL            DUSHEY   

               Owner's Business Name  City   Owner'sPhone #  \
8                     CLARITY 47 LLC    NaN      2124218100   
19          2065 PROPERTY OWNERS LLC    NaN      2126794590   
26  SAGAMORE AK PROPERTY COMPANY LLC    NaN      2124526077   
27                BP UM PARTNERS LLC    NaN      9172945840   
33         540 FULTON ASSOCIATES LLC    NaN      2128896405   

            Job Description           DOBRunDate  JOB_S1_NO  GIS_LATITUDE  \
8                       NaN  12/28/2021 00:00:00    2765418     40.758577   
19                      NaN  08/21/2021 00:00:00    2858979     40.852175   
26  NEW 14TH STORY BUILDING  12/02/2021 00:00:00    2877572     40.722048   
27                      NaN  07/31/2021 00:00:00    2936660     40.827907   
33                      NaN  05/07/2021 00:00:00    2605515     40.689011   

    GIS_LONGITUDE                                      GIS_NTA_NAME  
8      -73.983100                             Midtown-Midtown South  
19     -73.901116                                        Mount Hope  
26     -73.9918

In [34]:
import csv 
mrkt_over_10_2019.to_csv("mrkt_over_10_2019")

In [35]:
mrkt_over_10_2019['Proposed No. of Stories'].unique()

array([48, 12, 14, 43, 23, 10, 11, 15, 17, 19, 66, 22, 26, 33, 27, 56, 25,
       21, 20, 35, 29, 16, 13, 38, 37, 74, 18, 24, 46, 60])

In [36]:
mrkt_over_10_2019["Fully Permitted"].min()

Timestamp('2019-01-02 00:00:00')

In [37]:
mrkt_over_10_2019["Fully Permitted"].max()

Timestamp('2019-12-26 00:00:00')

In [38]:
index = mrkt_over_10_2019.index
number_of_rows = len(index)
print(number_of_rows)

115


In [39]:
over_seven_2019 = nineteen_job_market['Proposed No. of Stories'] >= 7

In [40]:
under_9_2019 = nineteen_job_market['Proposed No. of Stories'] <= 9

In [41]:
btwn_7_and_9_2019 = over_seven_2019 & under_9_2019

In [42]:
mrkt_btwn_7_and_9_2019 = nineteen_job_market.loc[btwn_7_and_9_2019]

In [43]:
mrkt_btwn_7_and_9_2019['Proposed No. of Stories'].unique()

array([8, 9, 7])

In [44]:
mrkt_btwn_7_and_9_2019["Fully Permitted"].min()

Timestamp('2019-01-07 00:00:00')

In [45]:
mrkt_btwn_7_and_9_2019["Fully Permitted"].max()

Timestamp('2019-12-30 00:00:00')

In [46]:
mrkt_btwn_7_and_9_2019.to_csv("mrkt_btwn_7_and_9_2019")

In [47]:
index = mrkt_btwn_7_and_9_2019.index
number_of_rows = len(index)
print(number_of_rows)

133


- In 2019, there were a total of 248 fully permitted jobs designated as new buildings ("NB")
- 115 of those jobs were 10 stories or higher
- 133 of those jobs were between 7 and 9 stories high.

# Job Market 2020

In [48]:
start_date_2020 = "2020-01-01"

end_date_2020 = "2020-12-31"

after_start_date_2020 = df["Fully Permitted"] >= start_date_2020

before_end_date_2020 = df["Fully Permitted"] <= end_date_2020

full_year_2020 = after_start_date_2020 & before_end_date_2020

job_market_2020 = df.loc[full_year_2020]

job_market_2020.head()

Job #   Borough House #      Street Name Job Type Job Status  \
16  320978064  BROOKLYN    2286   CROPSEY AVENUE       NB          R   
22  420666032    QUEENS   29-17      40TH AVENUE       NB          R   
29  220699705     BRONX    2086    VALENTINE AVE       NB          R   
36  210178289     BRONX    2050  GRAND CONCOURSE       NB          R   
57  420664855    QUEENS   95-10       218 STREET       NB          R   

                  Job Status Descrp Latest Action Date Building Type  \
16  PERMIT ISSUED - ENTIRE JOB/WORK         12/21/2020        OTHERS   
22  PERMIT ISSUED - ENTIRE JOB/WORK         07/23/2021        OTHERS   
29  PERMIT ISSUED - ENTIRE JOB/WORK         10/07/2021        OTHERS   
36  PERMIT ISSUED - ENTIRE JOB/WORK         05/06/2021        OTHERS   
57  PERMIT ISSUED - ENTIRE JOB/WORK         05/11/2021        OTHERS   

    Community - Board Applicant's First Name Applicant's Last Name  \
16                311                MICHAEL                  KANG   
22                401            CHRISTOPHER               FOGARTY   
29                205                ANTHONY               DANIELS   
36                205                  VILLA              FERNANDO   
57                413                 MANISH                SAVANI   

   Applicant Professional Title  Applicant License # Professional Cert  \
16                           RA                24849                 N   
22                           RA                28860                 N   
29                           RA                26668                 N   
36                           RA               308141                 N   
57                           RA                29410                 N   

   Fully Permitted  Existing Zoning Sqft  Proposed Zoning Sqft  \
16      2020-12-21                     0                241259   
22      2020-05-20                     0                 70835   
29      2020-08-26                     0                 34259   
36      2020-04-17                     0                 63871   
57      2020-03-26                     0                 23919   

    ExistingNo. of Stories  Proposed No. of Stories  Existing Height  \
16                       0                       23                0   
22                       0                       10                0   
29                       0                        9                0   
36                       0                       13                0   
57                       0                        9                0   

    Proposed Height Owner's First Name Owner's Last Name  \
16              240           MENG HUA              WANG   
22              120                JOE             STERN   
29               80              CRAIG             NASSI   
36              135             YVETTE            ANDREL   
57               97           AMRITPAL            SANDHU   

          Owner's Business Name  City   Owner'sPhone #  \
16        CROPSEY PARTNERS LLC.    NaN      7189391002   
22                LIC OWNER LLC    NaN      7189371000   
29                 NOA LIRO LLC    NaN      9178482772   
36  2050 G.C. HOUSING DEV. FUND    NaN      7182317711   
57                218 ST NY LLC    NaN      7188467443   

                                      Job Description           DOBRunDate  \
16                                                NaN  05/04/2021 00:00:00   
22  CONSTRACTION OF NEW 10-STORY MIXED USE BUILDIN...  11/05/2021 00:00:00   
29                                                NaN  10/08/2021 00:00:00   
36                                                NaN  05/07/2021 00:00:00   
57  THIS APPLICATION IS FILED FOR NEW 9 STORY TRAN...  05/12/2021 00:00:00   

    JOB_S1_NO  GIS_LATITUDE  GIS_LONGITUDE  \
16    2354982     40.595843     -73.997001   
22    3127913     40.752206     -73.935379   
29    3097745     40.852849     -73.899854   
36    2975909     40.852509     -73.903527   
57    3036153     40.717577     -73.7

In [49]:
job_market_2020["Fully Permitted"].min()

Timestamp('2020-01-03 00:00:00')

In [50]:
job_market_2020["Fully Permitted"].max()

Timestamp('2020-12-30 00:00:00')

In [51]:
index = job_market_2020.index
number_of_rows = len(index)
print(number_of_rows)

218


In [136]:
# designate coordinate system
crs=CRS('EPSG:4326')
# zip x and y coordinates into single feature
geometry = [Point(xy) for xy in zip(job_market_2020['GIS_LONGITUDE'], job_market_2020['GIS_LATITUDE'])]
# create GeoPandas dataframe
geo_job_market_2020 = gpd.GeoDataFrame(job_market_2020,
 crs = crs,
 geometry = geometry)

In [137]:
geo_job_market_2020.to_file("geo_job_market_2020.geojson", driver='GeoJSON')

In [52]:
# Jobs 10 stories and over 2020. 

In [53]:
over_10_2020 = job_market_2020['Proposed No. of Stories'] >= 10

In [54]:
mrkt_over_10_2020 = job_market_2020.loc[over_10_2020]

In [55]:
mrkt_over_10_2020.head()

Job #    Borough House #      Street Name Job Type Job Status  \
16  320978064   BROOKLYN    2286   CROPSEY AVENUE       NB          R   
22  420666032     QUEENS   29-17      40TH AVENUE       NB          R   
36  210178289      BRONX    2050  GRAND CONCOURSE       NB          R   
62  121204026  MANHATTAN     566    BROOME STREET       NB          R   
76  320911572   BROOKLYN      18     SIXTH AVENUE       NB          R   

                  Job Status Descrp Latest Action Date Building Type  \
16  PERMIT ISSUED - ENTIRE JOB/WORK         12/21/2020        OTHERS   
22  PERMIT ISSUED - ENTIRE JOB/WORK         07/23/2021        OTHERS   
36  PERMIT ISSUED - ENTIRE JOB/WORK         05/06/2021        OTHERS   
62  PERMIT ISSUED - ENTIRE JOB/WORK         09/27/2021        OTHERS   
76  PERMIT ISSUED - ENTIRE JOB/WORK         08/20/2020        OTHERS   

    Community - Board Applicant's First Name Applicant's Last Name  \
16                311                MICHAEL                  KANG   
22                401            CHRISTOPHER               FOGARTY   
36                205                  VILLA              FERNANDO   
62                102                  SITAL                 PATEL   
76                302                   CARL              ORDEMANN   

   Applicant Professional Title  Applicant License # Professional Cert  \
16                           RA                24849                 N   
22                           RA                28860                 N   
36                           RA               308141                 N   
62                           RA                28577                 N   
76                           RA                15995                 N   

   Fully Permitted  Existing Zoning Sqft  Proposed Zoning Sqft  \
16      2020-12-21                     0                241259   
22      2020-05-20                     0                 70835   
36      2020-04-17                     0                 63871   
62      2020-11-12                     0                208018   
76      2020-05-04                     0                808695   

    ExistingNo. of Stories  Proposed No. of Stories  Existing Height  \
16                       0                       23                0   
22                       0                       10                0   
36                       0                       13                0   
62                       0                       27                0   
76                       0                       51                0   

    Proposed Height Owner's First Name Owner's Last Name  \
16              240           MENG HUA              WANG   
22              120                JOE             STERN   
36              135             YVETTE            ANDREL   
62              290              LOUIS           MADIGAN   
76              525                JON           ADAMSKI   

          Owner's Business Name  City   Owner'sPhone #  \
16        CROPSEY PARTNERS LLC.    NaN      7189391002   
22                LIC OWNER LLC    NaN      7189371000   
36  2050 G.C. HOUSING DEV. FUND    NaN      7182317711   
62     MADIGAN DEVELOPMENT, LLC    NaN      2123151505   
76    18 SIXTH AVENUE OWNER LLC    NaN      2128965576   

                                      Job Description           DOBRunDate  \
16                                                NaN  05/04/2021 00:00:00   
22  CONSTRACTION OF NEW 10-STORY MIXED USE BUILDIN...  11/05/2021 00:00:00   
36                                                NaN  05/07/2021 00:00:00   
62                                                NaN  10/19/2021 00:00:00   
76           ERECT 51 STORY NEW BUILDING WITH CELLAR.  05/18/2021 00:00:00   

    JOB_S1_NO  GIS_LATITUDE  GIS_LONGITUDE  \
16    2354982     40.595843     -73.997001   
22    3127913     40.752206     -73.935379   
36    2975909     40.852509     -73.903527   
62    2833270     40.724447     -74.006418   
76    3013619     40.682452    

In [56]:
mrkt_over_10_2020.to_csv("mrkt_over_10_2020")

In [57]:
mrkt_over_10_2020['Proposed No. of Stories'].unique()

array([23, 10, 13, 27, 51, 18, 43, 35, 38, 12, 19, 29, 52, 11, 22, 34, 24,
       28, 14, 16, 31, 30, 25, 26, 60, 15, 17, 33, 57, 40, 44, 20])

In [58]:
index = mrkt_over_10_2020.index
number_of_rows = len(index)
print(number_of_rows)

77


In [140]:
# designate coordinate system
crs=CRS('EPSG:4326')
# zip x and y coordinates into single feature
geometry = [Point(xy) for xy in zip(mrkt_over_10_2020['GIS_LONGITUDE'], mrkt_over_10_2020['GIS_LATITUDE'])]
# create GeoPandas dataframe
geo_mrkt_over_10_2020 = gpd.GeoDataFrame(mrkt_over_10_2020,
 crs = crs,
 geometry = geometry)

In [141]:
geo_mrkt_over_10_2020.to_file("geo_mrkt_over_10_2020.geojson", driver='GeoJSON')

In [59]:
#Jobs between 7 and 9 stories 2020

In [60]:
over_7_2020 = job_market_2020['Proposed No. of Stories'] >= 7

In [61]:
under_9_2020 = job_market_2020['Proposed No. of Stories'] <= 9

In [62]:
bwtn_7_and_9_2020 = over_7_2020 & under_9_2020

In [63]:
mrkt_bwtn_7_and_9_2020 = job_market_2020.loc[bwtn_7_and_9_2020]

In [64]:
mrkt_bwtn_7_and_9_2020['Proposed No. of Stories'].unique()

array([9, 7, 8])

In [65]:
mrkt_bwtn_7_and_9_2020.head()

Job #   Borough House #      Street Name Job Type Job Status  \
29  220699705     BRONX    2086    VALENTINE AVE       NB          R   
57  420664855    QUEENS   95-10       218 STREET       NB          R   
72  420666087    QUEENS  188-11  HILLSIDE AVENUE       NB          R   
83  321745849  BROOKLYN     484      STERLING PL       NB          R   
85  421133320    QUEENS   26-41           3RD ST       NB          R   

                  Job Status Descrp Latest Action Date Building Type  \
29  PERMIT ISSUED - ENTIRE JOB/WORK         10/07/2021        OTHERS   
57  PERMIT ISSUED - ENTIRE JOB/WORK         05/11/2021        OTHERS   
72  PERMIT ISSUED - ENTIRE JOB/WORK         07/09/2021        OTHERS   
83  PERMIT ISSUED - ENTIRE JOB/WORK         03/19/2021        OTHERS   
85  PERMIT ISSUED - ENTIRE JOB/WORK         12/23/2020        OTHERS   

    Community - Board Applicant's First Name Applicant's Last Name  \
29                205                ANTHONY               DANIELS   
57                413                 MANISH                SAVANI   
72                408                 VICTOR           MIRONTSCHUK   
83                308                 ROBERT            LITCHFIELD   
85                401                 NARESH               MAHANGU   

   Applicant Professional Title  Applicant License # Professional Cert  \
29                           RA                26668                 N   
57                           RA                29410                 N   
72                           RA                19338                 N   
83                           RA                14807                 N   
85                           PE                89068                 N   

   Fully Permitted  Existing Zoning Sqft  Proposed Zoning Sqft  \
29      2020-08-26                     0                 34259   
57      2020-03-26                     0                 23919   
72      2020-10-26                     0                 72308   
83      2020-01-07                     0                 35949   
85      2020-12-23                     0                 54906   

    ExistingNo. of Stories  Proposed No. of Stories  Existing Height  \
29                       0                        9                0   
57                       0                        9                0   
72                       0                        7                0   
83                       0                        7                0   
85                       0                        7                0   

    Proposed Height Owner's First Name Owner's Last Name  \
29               80              CRAIG             NASSI   
57               97           AMRITPAL            SANDHU   
72               70            MICHAEL              GREY   
83               72            WILLIAM             CALEO   
85               75           YITZCHOK              KATZ   

           Owner's Business Name  City   Owner'sPhone #  \
29                  NOA LIRO LLC    NaN      9178482772   
57                 218 ST NY LLC    NaN      7188467443   
72  188-11 HILLSIDE PARTNERS LLC    NaN      5162222233   
83           484 STERLING PL LLC    NaN      7187150418   
85      DEVELOPING NY STATE, LLC    NaN      7182463734   

                                      Job Description           DOBRunDate  \
29                                                NaN  10/08/2021 00:00:00   
57  THIS APPLICATION IS FILED FOR NEW 9 STORY TRAN...  05/12/2021 00:00:00   
72                                                NaN  12/30/2021 00:00:00   
83  CONSTRUCTION OF NEW 7 STORY BUILDING WITH 25 D...  05/20/2021 00:00:00   
85                                                NaN  10/12/2021 00:00:00   

    JOB_S1_NO  GIS_LATITUDE  GIS_LONGITUDE                GIS_NTA_NAME  
29    3097745     40.852849     -73.899854                  Mount Hope  
57    3036153     40.717577     -73.737406              Queens Village  
72    3135320     40.715188     -73.773394

In [66]:
mrkt_bwtn_7_and_9_2020.to_csv("mrkt_bwtn_7_and_9_2020")

In [67]:
index = mrkt_bwtn_7_and_9_2020.index
number_of_rows = len(index)
print(number_of_rows)

141


In [142]:
# designate coordinate system
crs=CRS('EPSG:4326')
# zip x and y coordinates into single feature
geometry = [Point(xy) for xy in zip(mrkt_bwtn_7_and_9_2020['GIS_LONGITUDE'], mrkt_bwtn_7_and_9_2020['GIS_LATITUDE'])]
# create GeoPandas dataframe
geo_mrkt_bwtn_7_and_9_2020 = gpd.GeoDataFrame(mrkt_bwtn_7_and_9_2020,
 crs = crs,
 geometry = geometry)

In [143]:
geo_mrkt_bwtn_7_and_9_2020.to_file("geo_mrkt_bwtn_7_and_9_2020.geojson", driver='GeoJSON')

- In 2020, there were a total of 218 fully permitted jobs designated as new buildings ("NB")
- 77 of those jobs were 10 stories or higher
- 141 of those jobs were between 7 and 9 stories high.

# Job Market 2021

In [68]:
start_date_2021 = "2021-01-01"

end_date_2021 = "2021-12-31"

after_start_date_2021 = df["Fully Permitted"] >= start_date_2021

before_end_date_2021 = df["Fully Permitted"] <= end_date_2021

full_year_2021 = after_start_date_2021 & before_end_date_2021

job_market_2021 = df.loc[full_year_2021]

job_market_2021.head()

Job #    Borough House #      Street Name Job Type Job Status  \
0  220538611      BRONX     915       ELSMERE PL       NB          R   
2  220649706      BRONX    3923    CARPENTER AVE       NB          R   
3  121205604  MANHATTAN     270      PARK AVENUE       NB          R   
4  321597795   BROOKLYN     963  ATLANTIC AVENUE       NB          R   
5  321941182   BROOKLYN    1882         BROADWAY       NB          R   

                 Job Status Descrp Latest Action Date Building Type  \
0  PERMIT ISSUED - ENTIRE JOB/WORK         11/05/2021        OTHERS   
2  PERMIT ISSUED - ENTIRE JOB/WORK         04/29/2021        OTHERS   
3  PERMIT ISSUED - ENTIRE JOB/WORK         11/30/2021        OTHERS   
4  PERMIT ISSUED - ENTIRE JOB/WORK         06/22/2021        OTHERS   
5  PERMIT ISSUED - ENTIRE JOB/WORK         05/05/2021        OTHERS   

   Community - Board Applicant's First Name Applicant's Last Name  \
0                206                 MINKUK                   KIM   
2                212                  JULIO            LEDER-LUIS   
3                105               NICKOLAS             ZIGOMANIS   
4                302                 DANIEL             BERNSTEIN   
5                316                JEFFREY                 KAMEN   

  Applicant Professional Title  Applicant License # Professional Cert  \
0                           RA                32897                 N   
2                           RA                20478                 N   
3                           RA                33081                 N   
4                           RA                16854                 N   
5                           RA                23279                 N   

  Fully Permitted  Existing Zoning Sqft  Proposed Zoning Sqft  \
0      2021-04-29                     0                 49075   
2      2021-03-16                     0                 20237   
3      2021-11-30                     0               1864801   
4      2021-06-22                     0                 88368   
5      2021-05-05                     0                 63644   

   ExistingNo. of Stories  Proposed No. of Stories  Existing Height  \
0                       0                        9                0   
2                       0                        7                0   
3                       0                       57                0   
4                       0                        9                0   
5                       0                        7                0   

   Proposed Height Owner's First Name Owner's Last Name  \
0               90                JAY           MARTINO   
2               65             ROBERT             LUMAJ   
3             1388              DEVIN             MAHER   
4               95             EFRAIM             WEBER   
5               70            YISROEL         GREENFELD   

            Owner's Business Name  City   Owner'sPhone #  \
0                     STAGG GROUP    NaN      9146687511   
2        ATLANTIS RENOVATIONS INC    NaN      3476210292   
3  JP MORGAN AND CHASE BANK, N.A.    NaN      2122704958   
4                86 LEFFERTS CORP    NaN      9176824871   
5               BROADWAY 1884 LLC    NaN      3476755480   

                             Job Description           DOBRunDate  JOB_S1_NO  \
0  ERECTION OF 9 STORY RESIDENTIAL BUILDING.  11/23/2021 00:00:00    2637000   
2                                        NaN  04/30/2021 00:00:00    2965629   
3                                        NaN  12/09/2021 00:00:00    3126397   
4                                        NaN  10/12/2021 00:00:00    3158207   
5                                        NaN  01/05/2022 00:00:00    3161454   

   GIS_LATITUDE  GIS_LONGITUDE              GIS_NTA_NAME  
0     40.841590     -73.885100              East Tremont  
2     40.887420     -73.864069  Williamsbridge-Olinville  
3     40.755641     -73.975109     Midtown-Midtown South  
4     40.680263     -73.960337            

In [69]:
job_market_2021["Fully Permitted"].min()

Timestamp('2021-01-05 00:00:00')

In [70]:
job_market_2021["Fully Permitted"].max()

Timestamp('2021-12-30 00:00:00')

In [71]:
index = job_market_2021.index
number_of_rows = len(index)
print(number_of_rows)

231


In [138]:
# designate coordinate system
crs=CRS('EPSG:4326')
# zip x and y coordinates into single feature
geometry = [Point(xy) for xy in zip(job_market_2021['GIS_LONGITUDE'], job_market_2021['GIS_LATITUDE'])]
# create GeoPandas dataframe
geo_job_market_2021 = gpd.GeoDataFrame(job_market_2021,
 crs = crs,
 geometry = geometry)

In [139]:
geo_job_market_2021.to_file("geo_job_market_2021.geojson", driver='GeoJSON')

In [72]:
# Jobs 10 stories and over 2021. 

In [73]:
over_10_2021 = job_market_2021['Proposed No. of Stories'] >= 10

In [74]:
mrkt_over_10_2021 = job_market_2021.loc[over_10_2021]

In [159]:
mrkt_over_10_2021.head()

Job #    Borough House #      Street Name Job Type Job Status  \
3   121205604  MANHATTAN     270      PARK AVENUE       NB          R   
6   210182283      BRONX    2351    WALTON AVENUE       NB          R   
10  321588545   BROOKLYN      34        UNION AVE       NB          R   
20  121188491  MANHATTAN      50  EAST 112 STREET       NB          R   
21  421644787     QUEENS   35-32   LEAVITT STREET       NB          R   

                  Job Status Descrp Latest Action Date Building Type  \
3   PERMIT ISSUED - ENTIRE JOB/WORK         11/30/2021        OTHERS   
6   PERMIT ISSUED - ENTIRE JOB/WORK         10/14/2021        OTHERS   
10  PERMIT ISSUED - ENTIRE JOB/WORK         04/30/2021        OTHERS   
20  PERMIT ISSUED - ENTIRE JOB/WORK         09/15/2021        OTHERS   
21  PERMIT ISSUED - ENTIRE JOB/WORK         12/13/2021        OTHERS   

    Community - Board Applicant's First Name Applicant's Last Name  \
3                 105               NICKOLAS             ZIGOMANIS   
6                 205                  DAVID                 GROSS   
10                301                 FARIBA                MAKOOI   
20                111                  DAVID             MIDDLETON   
21                407                RAYMOND                  CHAN   

   Applicant Professional Title  Applicant License # Professional Cert  \
3                            RA                33081                 N   
6                            RA                16587                 N   
10                           RA                34686                 N   
20                           RA                19933                 N   
21                           RA                15817                 N   

   Fully Permitted  Existing Zoning Sqft  Proposed Zoning Sqft  \
3       2021-11-30                     0               1864801   
6       2021-10-14                     0                 28585   
10      2021-04-30                     0                423295   
20      2021-09-15                     0                655936   
21      2021-09-16                     0                152161   

    ExistingNo. of Stories  Proposed No. of Stories  Existing Height  \
3                        0                       57                0   
6                        0                       12                0   
10                       0                       12                0   
20                       0                       34                0   
21                       0                       13                0   

    Proposed Height Owner's First Name Owner's Last Name  \
3              1388              DEVIN             MAHER   
6               114             STEVEN             MOUNT   
10              114              SIMON         DUSHINSKY   
20              398            MICHAEL              DALY   
21              152             GEORGE                XU   

             Owner's Business Name  City   Owner'sPhone # Job Description  \
3   JP MORGAN AND CHASE BANK, N.A.    NaN      2122704958             NaN   
6            2351 WALTON OWNER LLC    NaN      9172177359             NaN   
10             HARRISON REALTY LLC    NaN      7182464762             NaN   
20                 SV-A OWNERS LLC    NaN      9175423600             NaN   
21             CG & J REALTY, LLC.    NaN      7189391060             NaN   

             DOBRunDate  JOB_S1_NO  GIS_LATITUDE  GIS_LONGITUDE  \
3   12/09/2021 00:00:00    3126397     40.755641     -73.975109   
6   10/15/2021 00:00:00    3270079     40.859644     -73.902027   
10  10/06/2021 00:00:00    3193905     40.700831     -73.950274   
20  11/19/2021 00:00:00    2959650     40.797129     -73.945972   
21  12/14/2021 00:00:00    3000608     40.764245     -73.828835   

             GIS_NTA_NAME                    geometry  
3   Midtown-Midtown South  POINT (-73.97511 40.75564)  
6           Fordham South  POINT (-73.90203 40.85964)  
10                Bedford  POINT (-73.95027 4

In [160]:
mrkt_over_10_2021['Proposed No. of Stories'].unique()

array([57, 12, 34, 13, 14, 16, 17, 11, 31, 18, 24, 23, 22, 10, 51, 20, 15,
       19, 40, 27, 30, 32, 28])

In [161]:
index = mrkt_over_10_2021.index
number_of_rows = len(index)
print(number_of_rows)

75


In [162]:
# designate coordinate system
crs=CRS('EPSG:4326')
# zip x and y coordinates into single feature
geometry = [Point(xy) for xy in zip(mrkt_over_10_2021['GIS_LONGITUDE'], mrkt_over_10_2021['GIS_LATITUDE'])]
# create GeoPandas dataframe
geo_mrkt_over_10_2021 = gpd.GeoDataFrame(mrkt_over_10_2021,
 crs = crs,
 geometry = geometry)

In [163]:
geo_mrkt_over_10_2021.to_file("geo_mrkt_over_10_2021.geojson", driver='GeoJSON')

In [78]:
# Jobs bewteen 7 and 9 stories in 2021.

In [79]:
over_7_2021 = job_market_2021['Proposed No. of Stories'] >= 7

In [80]:
under_9_2021 = job_market_2021['Proposed No. of Stories'] <= 9

In [81]:
bwtn_7_and_9_2021 = over_7_2021 & under_9_2021

In [82]:
mrkt_bwtn_7_and_9_2021 = job_market_2021.loc[bwtn_7_and_9_2021]

In [83]:
mrkt_bwtn_7_and_9_2021['Proposed No. of Stories'].unique()

array([9, 7, 8])

In [155]:
mrkt_bwtn_7_and_9_2021.head()

Job #   Borough House #        Street Name Job Type Job Status  \
0  220538611     BRONX     915         ELSMERE PL       NB          R   
2  220649706     BRONX    3923      CARPENTER AVE       NB          R   
4  321597795  BROOKLYN     963    ATLANTIC AVENUE       NB          R   
5  321941182  BROOKLYN    1882           BROADWAY       NB          R   
7  421133794    QUEENS   71-82  PARSONS BOULEVARD       NB          R   

                 Job Status Descrp Latest Action Date Building Type  \
0  PERMIT ISSUED - ENTIRE JOB/WORK         11/05/2021        OTHERS   
2  PERMIT ISSUED - ENTIRE JOB/WORK         04/29/2021        OTHERS   
4  PERMIT ISSUED - ENTIRE JOB/WORK         06/22/2021        OTHERS   
5  PERMIT ISSUED - ENTIRE JOB/WORK         05/05/2021        OTHERS   
7  PERMIT ISSUED - ENTIRE JOB/WORK         08/30/2021        OTHERS   

   Community - Board Applicant's First Name Applicant's Last Name  \
0                206                 MINKUK                   KIM   
2                212                  JULIO            LEDER-LUIS   
4                302                 DANIEL             BERNSTEIN   
5                316                JEFFREY                 KAMEN   
7                408                 ROBERT             BIANCHINI   

  Applicant Professional Title  Applicant License # Professional Cert  \
0                           RA                32897                 N   
2                           RA                20478                 N   
4                           RA                16854                 N   
5                           RA                23279                 N   
7                           RA                33616                 N   

  Fully Permitted  Existing Zoning Sqft  Proposed Zoning Sqft  \
0      2021-04-29                     0                 49075   
2      2021-03-16                     0                 20237   
4      2021-06-22                     0                 88368   
5      2021-05-05                     0                 63644   
7      2021-08-30                     0                 74621   

   ExistingNo. of Stories  Proposed No. of Stories  Existing Height  \
0                       0                        9                0   
2                       0                        7                0   
4                       0                        9                0   
5                       0                        7                0   
7                       0                        8                0   

   Proposed Height Owner's First Name Owner's Last Name  \
0               90                JAY           MARTINO   
2               65             ROBERT             LUMAJ   
4               95             EFRAIM             WEBER   
5               70            YISROEL         GREENFELD   
7               85             JOSHUA          ASHERIAN   

      Owner's Business Name  City   Owner'sPhone #  \
0               STAGG GROUP    NaN      9146687511   
2  ATLANTIS RENOVATIONS INC    NaN      3476210292   
4          86 LEFFERTS CORP    NaN      9176824871   
5         BROADWAY 1884 LLC    NaN      3476755480   
7         157-01 72 AVE LLC    NaN      2122730794   

                             Job Description           DOBRunDate  JOB_S1_NO  \
0  ERECTION OF 9 STORY RESIDENTIAL BUILDING.  11/23/2021 00:00:00    2637000   
2                                        NaN  04/30/2021 00:00:00    2965629   
4                                        NaN  10/12/2021 00:00:00    3158207   
5                                        NaN  01/05/2022 00:00:00    3161454   
7                                        NaN  08/31/2021 00:00:00    3280040   

   GIS_LATITUDE  GIS_LONGITUDE                        GIS_NTA_NAME  \
0     40.841590     -73.885100                        East Tremont   
2     40.887420     -73.864069            Williamsbridge-Olinville   
4     40.680263     -73.960337                        Clinton Hill   
5     40.680622     -73.9064

In [156]:
index = mrkt_bwtn_7_and_9_2021.index
number_of_rows = len(index)
print(number_of_rows)

156


In [157]:
# designate coordinate system
crs=CRS('EPSG:4326')
# zip x and y coordinates into single feature
geometry = [Point(xy) for xy in zip(mrkt_bwtn_7_and_9_2021['GIS_LONGITUDE'], mrkt_bwtn_7_and_9_2021['GIS_LATITUDE'])]
# create GeoPandas dataframe
geo_mrkt_bwtn_7_and_9_2021 = gpd.GeoDataFrame(mrkt_bwtn_7_and_9_2021,
 crs = crs,
 geometry = geometry)

In [158]:
geo_mrkt_bwtn_7_and_9_2021.to_file("mrkt_bwtn_7_and_9_2021.geojson", driver='GeoJSON')

- In 2021, there were a total of 231 fully permitted jobs designated as new buildings ("NB")
- 75 of those jobs were 10 stories or higher
- 156 of those jobs were between 7 and 9 stories high.

# Main Findings

- In 2019-2021, there were a total of 697 fully permitted jobs designated as new buildings ("NB")
- 267 of those jobs were 10 stories or higher
- 430 of those jobs were between 7 and 9 stories high.

- In 2019, there were a total of 248 fully permitted jobs designated as new buildings ("NB")
- 115 of those jobs were 10 stories or higher
- 133 of those jobs were between 7 and 9 stories high.

- In 2020, there were a total of 218 fully permitted jobs designated as new buildings ("NB")
- 77 of those jobs were 10 stories or higher
- 141 of those jobs were between 7 and 9 stories high.   

- In 2021, there were a total of 231 fully permitted jobs designated as new buildings ("NB")
- 75 of those jobs were 10 stories or higher
- 156 of those jobs were between 7 and 9 stories high.